### PR TITLE
develop: update peewee guide

### PR DESCRIPTION
**Pull request by wd0517**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Oreoxmt — APPROVED** [review]

File: develop/dev-guide-sample-application-python-peewee.md [hunk 1]
```diff
@@ -1,245 +1,294 @@
...
 ```
 
-此处将以 peewee **3.15.4** 版本进行说明。
+#### 为什么安装 PyMySQL？
 
-```python
-import os
-import uuid
-from typing import List
+Peewee 是一个支持多种数据库的 ORM 库。它是对数据库的高层抽象，可以帮助开发者以更面向对象的方式编写 SQL 语句。但 peewee 并不提供数据库驱动，因此需要单独安装用于连接 TiDB 的驱动。本示例项目使用 PyMySQL 作为数据库驱动。PyMySQL 是一个与 TiDB 兼容的纯 Python 实现的 MySQL 客户端库，并可以在所有平台上安装。更多信息，参考 [peewee 官方文档](https://docs.peewee-orm.com/en/latest/peewee/database.html?highlight=mysql#using-mysql)。
```
[the remark this answers is not in the document]
Comment (wd0517) [reply develop/dev-guide-sample-application-python-peewee.md]: peewee 是第一个单词的时候它就是大写的，它的官方文档也是这样 http://docs.peewee-orm.com/en/latest/

**Review by Icemap — APPROVED** [review]

File: develop/dev-guide-sample-application-python-peewee.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Icemap) [reply develop/dev-guide-sample-application-python-peewee.md]: 哦哦，好的，那我改回来哈。

File: develop/dev-guide-sample-application-python-peewee.md [hunk 2]
```diff
@@ -1,245 +1,294 @@
...
 ```
 
-此处将以 peewee **3.15.4** 版本进行说明。
+#### 为什么安装 PyMySQL？
 
-```python
-import os
-import uuid
-from typing import List
+peewee 是一个支持多种数据库的 ORM 库。它是对数据库的高层抽象，可以帮助开发者以更面向对象的方式编写 SQL 语句。但 peewee 并不提供数据库驱动，因此需要单独安装用于连接 TiDB 的驱动。本示例项目使用 PyMySQL 作为数据库驱动。PyMySQL 是一个与 TiDB 兼容的纯 Python 实现的 MySQL 客户端库，并可以在所有平台上安装。更多信息，参考 [peewee 官方文档](https://docs.peewee-orm.com/en/latest/peewee/database.html?highlight=mysql#using-mysql)。
```
Comment (Icemap) [inline develop/dev-guide-sample-application-python-peewee.md:51]:
```suggestion
Peewee 是一个支持多种数据库的 ORM 库。它是对数据库的高层抽象，可以帮助开发者以更面向对象的方式编写 SQL 语句。但 peewee 并不提供数据库驱动，因此需要单独安装用于连接 TiDB 的驱动。本示例项目使用 PyMySQL 作为数据库驱动。PyMySQL 是一个与 TiDB 兼容的纯 Python 实现的 MySQL 客户端库，并可以在所有平台上安装。更多信息，参考 [peewee 官方文档](https://docs.peewee-orm.com/en/latest/peewee/database.html?highlight=mysql#using-mysql)。
```

File: develop/dev-guide-sample-application-python-peewee.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Icemap) [reply develop/dev-guide-sample-application-python-peewee.md]: 啊，已经 Merge 了……

[the remark this answers is not in the document]
Comment (Icemap) [reply develop/dev-guide-sample-application-python-peewee.md]: 需要的话，再水一个 PR 吧 XD

**Pull request merged**